spec/lib/traject/config/callnum_facet_spec.rb

```diff
@@ -325,13 +260,9 @@ def record_with_999(call_number:, scheme:, indexer:, home_location: 'STACKS', li
         # Harvard Yenching call numbers
         '6.4C-CZ[BC]',
         '2345 5861 V.3',
-        '2061 4246 NO.5-6 1936-1937',
```

Comment:
Removed these 4 because they had schemes that we don't process into callnum_facet_hsim